raiden/api/python.py

```diff
@@ -146,21 +146,22 @@ def leave_token_network(self, token_address, only_receiving=True):
         connection_manager = self.raiden.connection_manager_for_token(token_address)
         return connection_manager.leave(only_receiving)
 
-    def get_connection_manager_funds(self, token_address):
-        """Get the connection manager for a specific token"""
-        connection_manager = self.raiden.connection_manager_for_token(token_address)
-        if not connection_manager:
-            return None
-        else:
-            return connection_manager.funds
-
-    def get_connection_managers_list(self):
-        connection_managers = []
+    def get_connection_managers_info(self):
+        """Get a dict, indexed by token addresses, of connection managers
+        with open channels, containing funds, sum_deposits and number of channels"""
+        connection_managers = {}
```

Comment:
when we intiialize an empty dict in the rest of the code we usually use `dict()`. Please change to that form.